### PR TITLE
fix: fix some issues

### LINKED ISSUE #75: 表达式缺少右括号
https://github.com/VickScarlet/lifeRestart/blob/3c42cfb54633d5562ebdb547d1f53e7c7718b9c4/data/events.json#L616

### LINKED ISSUE #95: 属性选择界面建议添加user-select: none防止选中文字
如题
加点界面点击过快容易选中文本

### PR DESCRIPTION
**Pull request by liby**
## Specific changes

1. Fixed #75 
2. Fixed #95 
3. Add local development configuration

### CODE REVIEW

File: webpack.config.cjs [hunk 1]
```diff
@@ -5,7 +5,7 @@ module.exports = {
   mode: 'production',
   output: {
     path: path.resolve(__dirname, 'public'),
-    filename: 'bundle.js',
+    filename: '[contenthash].bundle.js',
```
Comment (SukkaW) [inline webpack.config.cjs:8]: Nope. `index.html` has a hardcoded `bundle.js` path and IMHO it is not the best practice to introduce dynamic `index.html` generation.

Comment (liby) [reply webpack.config.cjs:8]: You’re right. I missed it

**Comment by liby** [comment]
@SukkaW 
PTAL. Is this PR doing too much? In fact, I think Vite is more suitable for this repo than webpack.

File: view/style.css [hunk 2]
```diff
@@ -1,185 +1,334 @@
...
-@media (min-width:391px) and (max-width:399px){html{font-size:14.66px;}}
-@media (min-width:382px) and (max-width:390px){html{font-size:14.33px;}}
-@media (min-width:374px) and (max-width:381px){html{font-size:14px;}}
-@media (min-width:365px) and (max-width:373px){html{font-size:13.66px;}}
-@media (min-width:356px) and (max-width:364px){html{font-size:13.33px;}}
-@media (min-width:347px) and (max-width:355px){html{font-size:13px;}}
-@media (min-width:338px) and (max-width:346px){html{font-size:12.66px;}}
-@media (min-width:329px) and (max-width:337px){html{font-size:12.44px;}}
-@media (max-width:328px){html{font-size:12px;}}
+@media (min-width: 640px) {
```
Comment (BeautyYuYanli) [inline view/style.css:1]: 大规模的格式化可能给其它 contribute 产生巨量 conflict?

Comment (liby) [reply view/style.css]: Sorry, this should be caused by the editor auto-formatting when the file is saved. PTAL.

**Pull request merged**